CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Improved branch prediction hints on stable. (#209)
 - Optimized hashing of primitive types with AHash using specialization. (#207)
 - Only instantiate `RawTable`'s reserve functions once per key-value. (#204)
+- The minimum Rust version has been bumped to 1.38 (due to use of
```

Comment:
This should be in the "Unreleased" section.

Reply:
Oh, I was confused because I didn't see a git tag for v0.10.0, but I guess that release was yanked.

Reply:
Oops, I forgot to push the tag. (fixed)